Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-current-contest.tsx

```diff
@@ -144,11 +153,26 @@ const CurrentContestsProvider = ({ children }: ICurrentContestsProviderProps) =>
             contest: newContest,
             contestIsCompete,
             remainingTimeInMilliseconds: newRemainingTimeInMilliseconds,
-        } = responseData;
+            validationError: {
+                contestIsFound,
+                contestIsNotExpired,
+                contestCanBePracticed,
+                contestCanBeCompeted,
+                isParticipantRegistered,
+            },
```

Comment:
No need to decostruct, just to construct two line below. This is an example where a simple cast to `IStartParticipationValidationType` will be much better

Reply:
Fixed.